stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/SQLDataIterator.java

```diff
@@ -482,7 +483,7 @@ private void handleBooleanValue(MapValue<String, Object> bStruct, String fieldNa
     }
 
     private void handleDateValue(MapValue<String, Object> record, String fieldName, java.util.Date date,
-                                 BType fieldType) {
+                                 BType fieldType) throws ApplicationException {
```

Comment:
If we throw ApplicationException for the methods which should panic it will be a issue, as trap scenarios won't work properly. (This method is used by both remote methods and normal methods)

Reply:
Since I'm throwing from [1], I have changed.
[1] https://github.com/ballerina-platform/ballerina-lang/pull/16093/files#diff-19329368634f636768c0b86cba12a8f5R459